### DISCUSSION TITLE
How do I find library versions of delegates

### DISCUSSION THREAD

**Discussion by crohling** [ImageMagick/ImageMagick#6663] [Help]
### ImageMagick version

7.1.1

### Operating system, version and so on

Linux

### Description

Wondering how to figure out the verion of installed delegate libraries. Specifically I am looking to determine if the version of imagemagick I have installed has had it's libwebp delegate patched for `CVE-2023-4863`. Any help would be appreciated, thank you!

**Comment by snibgo** [comment]
```
magick -list format |grep -i webp
```
My result is:
```
             Compression options: None, Fax/Group3, Group4, JBIG, JPEG, LZW, LZMA, RLE, ZIP, ZSTD, WEBP
             Compression options: None, Fax/Group3, Group4, JBIG, JPEG, LZW, LZMA, RLE, ZIP, ZSTD, WEBP
             Compression options: None, Fax/Group3, Group4, JBIG, JPEG, LZW, LZMA, RLE, ZIP, ZSTD, WEBP
             Compression options: None, Fax/Group3, Group4, JBIG, JPEG, LZW, LZMA, RLE, ZIP, ZSTD, WEBP
     WEBP* rw+   WebP Image Format (libwebp 1.3.1 [020F])
```
So the libwebp used at IM's build was 1.3.1. I don't know how to trace libwebp patches.
